core/src/components/datetime-button/test/overlays/datetime-button.e2e.ts

```diff
@@ -159,4 +159,24 @@ test.describe('datetime-button: modal', () => {
     await ionModalDidPresent.next();
     expect(datetime).toBeVisible();
   });
+  test('datetime should be the first child of the modal', async ({ page }, testInfo) => {
```

Comment:
Can we move this test to the `modal` directory? The bug is modal-specific not datetime-specific.